src/throttler.guard.ts

```diff
@@ -72,28 +72,40 @@ export class ThrottlerGuard implements CanActivate {
         }
       }
     }
-    const tracker = this.getTracker(req);
-    const key = this.generateKey(context, tracker);
-    const ttls = await this.storageService.getRecord(key);
-    const nearestExpiryTime = ttls.length > 0 ? Math.ceil((ttls[0] - Date.now()) / 1000) : 0;
+    const t = this.getTracker(req);
```

Comment:
we should rename this variable. What about `tracker`?

Reply:
You are right, I revised in a new commit